packages/orchestration/src/fetched-chain-info.js

```diff
@@ -146,36 +148,36 @@ export default /** @type {const} } */ ({
     ],
     icqEnabled: false,
     connections: {
-      'neutron-1': {
-        id: 'connection-7',
-        client_id: '07-tendermint-29',
+      'osmosis-1': {
+        id: 'connection-2',
+        client_id: '07-tendermint-10',
         counterparty: {
-          client_id: '07-tendermint-48',
-          connection_id: 'connection-36',
+          client_id: '07-tendermint-3012',
+          connection_id: 'connection-2503',
         },
         state: 3,
         transferChannel: {
-          channelId: 'channel-8',
+          channelId: 'channel-2',
           portId: 'transfer',
-          counterPartyChannelId: 'channel-35',
+          counterPartyChannelId: 'channel-6994',
           counterPartyPortId: 'transfer',
           ordering: 0,
           state: 3,
           version: 'ics20-1',
         },
       },
-      'osmosis-1': {
-        id: 'connection-2',
-        client_id: '07-tendermint-10',
+      'neutron-1': {
```

Comment:
`yarn codegen` shouldn't create a diff like this (re-ordering of entries).

Need to audit the logic in `convertChainInfo`: https://github.com/Agoric/agoric-sdk/blob/7ae1f278fa8cbeb0cfc777b7cebf507b1f07c958/packages/orchestration/src/utils/registry.js#L120-L121

Reply:
See [cf708f8](https://github.com/Agoric/agoric-sdk/pull/10354/commits/cf708f8fe4f26adfd1e1361fb6d77799786e401d).